app/commands/markdown/parse.rb

```diff
@@ -8,12 +8,13 @@ def self.call(*args, **kwargs)
     new(*args, **kwargs).()
   end
 
-  def initialize(text, nofollow_links: false)
+  def initialize(text, nofollow_links: false, strip_h1: true) # rubocop:disable Naming/VariableNumber
     # TODO: We almost certainly don't want to do this!
     # but for now let's reduce the heading level of all
     # headings, as they're too high in the actual docs atm
     @text = text.to_s.gsub(/^##/, '###')
```

Comment:
I wondered if this would make more sense in the preprocessor.

Reply:
> I wondered if this would make more sense in the preprocessor.

I don't really understand where these different bits fit together now, so I'll be guided by you.

---

I suspect we want another flag which is `lower_heading_levels_by: 1`, which controls this. In different places, we'll want to drop headings by different amount of levels, now we've increased them all to h2 everywhere.

Reply:
Indeed. I think that makes sense.